doc/install_mne_c.rst

```diff
@@ -2,290 +2,175 @@
+- ``MATLAB_ROOT`` should give the path to your MATLAB binary (e.g.,
+  ``/opt/MATLAB/R2018b`` or similar).  If you do not have MATLAB or the MATLAB
+  runtime, leave ``MATLAB_ROOT`` undefined.
 
-To setup the MNE environment, a script must be sourced into your current shell.
-The result will be that additional environment variables like MNE_LIB_PATH
-will be set in your current shell in order to run the MNE Software.
-
-A note about source-ing scripts with "source" and "." (dot). The hash bang at
-the top of a script tells exec what shell to use by default to interpret the
-script. But the hash bang line is ignored when you source a script via either
-the builtin "source" or "." directive.  So there needs to be a setup script
-compatible with your current shell, i.e., if you use csh you cannot source a
-bash script.  (BTW, the "." (dot) directive will search for the script in
-$PATH if there is no slash in the script argument while "source" does not
-search $PATH).
-
-For Bourne or bash compatible shells, e.g., sh/bash/zsh, the script to source
-is ``$MNE_ROOT/bin/mne_setup_sh``.
-
-For C shells, e.g., csh/tcsh, the script to source
-is ``$MNE_ROOT/bin/mne_setup``.
-
-The SHELL variable should already be set in your current shell by your login or
-profile init file.  You can echo $SHELL on the command line to see what type of
-shell you are using with:
+Other environment variables are defined by setup scripts provided with MNE-C.
+You may either run the setup script each time you use MNE-C, or (recommended)
+configure your shell to run it automatically each time you open a terminal. For
+bash compatible shells, e.g., sh/bash/zsh, the script to source is
+``$MNE_ROOT/bin/mne_setup_sh``.  For C shells, e.g., csh/tcsh, the script to
+source is ``$MNE_ROOT/bin/mne_setup``.  If you don't know what shell you are
+using, you can run the following command to find out:
 
 .. code-block:: console
 
     $ echo $SHELL
 
-If the output indicates a POSIX shell ``bash`` or ``sh`` then you should export
-variables and source the bash/sh setup file with three commands:
+To configure MNE-C automatically for ``bash`` or ``sh`` shells, add this to
+your ``.bashrc``:
 
-.. code-block:: console
-
-    $ export MNE_ROOT=<MNE>
-    $ export MATLAB_ROOT=<Matlab>
-    $ . $MNE_ROOT/bin/mne_setup_sh
+.. code-block:: sh
 
-where ``<MNE>`` is replaced by the absolute path to where you have installed
-the MNE software. This path will be the parent path to the ./bin and ./lib
-subdirecotries in the MNE distribution. Or the command
-"ls $MNE_ROOT/bin/mne_setup_sh" should succeed if you have set MNE_ROOT
-correctly.
+    export MNE_ROOT=<path_to_MNE>
+    export MATLAB_ROOT=<path_to_MATLAB>
+    source $MNE_ROOT/bin/mne_setup_sh
 
-If you do not have a Matlab distribution, then leave MATLAB_ROOT undefined.
+where ``<path_to_MNE>`` and ``<path_to_MATLAB>`` are replaced by the absolute
+paths to MNE-C and MATLAB, respectively.
```

Comment:
I would repeat here that matlab is clearly optional.

Reply:
fixed in e10bdae